### PR TITLE
Includes ts files in coverage report

### PR DESCRIPTION
**Pull request by lewis-sanchez**
This PR includes ts files in the coverage report generated for tests.

[Hai Cao edit]:
Coverage task was initially failing in ADO pipeline with `TypeError: sourceMap.originalPositionFor is not a function` error.
Root cause: remap-istanbul was useing >=0.5.6 for the source-map package and later version of source-map included some break changes. Updating remap-istanbul to a newer version fixed this issue.
More info: https://github.com/SitePen/remap-istanbul/issues/160

TS files are still debuggable
image
image

### CODE REVIEW

**Comment by alanrenmsft** [comment]
the build is failing, please investigate.

File: package.json [hunk 1]
```diff
@@ -1129,5 +1129,11 @@
         }
       }
     }
+  },
+  "__metadata": {
```
Comment (alanrenmsft) [inline package.json:1133]: please remove this chunck

**Comment by alanrenmsft** [comment]
also add root cause analysis.

**Comment by caohai** [comment]
> also add root cause analysis.

Added related investigation, also included another screenshot for debugging in TS file.

**Review by alanrenmsft — APPROVED** [review]

**Pull request merged**